src/core/ui/palette/palette.ts

```diff
@@ -34,6 +34,7 @@ export const paletteOptions: PaletteOptions = {
   space: {
     dark: '#152A37',
   },
+  error: { main: '#940000' },
```

Comment:
Are you sure we need a new color for all errors? Maybe use the default one without setting this one and if needed we can change later after sync with the product team.